### DISCUSSION TITLE
Try Video Translation free hosted at VideoDubber.ai: AI Video Translation, Text-to-Speech, Subtitles Translation, Voice cloning

### DISCUSSION THREAD

**Discussion by Souvic** [facebookresearch/seamless_communication#290] [Show and tell]
# How can I translate a video?

[VideoDubber.ai](https://www.videodubber.com/) provides one-click video translation with voice cloning to top 30+ languages of the world by native speakers for free without any watermark.
So, just go to [VideoDubber.ai](https://www.videodubber.com/) and create an account to reach 10x more audience in their native language in minutes.

1. **Upload Your Video:** Use the uploader to import your video from Google Drive, Youtube, Twitter, Laptop, or Dropbox.

2. **Select Source and Target Language:** We support transcription in the top 30+ languages, dialects, and accents.

3. **Choose "Transcription" or "Subtitles":** Select "Subtitles" for a .srt file, or "Transcription" for a written transcript in plain text.

4. **Edit if you need:** Our automated transcription is quick and about 85% accurate. Opt for our human service for 99% accuracy by an expert native speaker.

5. **Receive Your Transcript/Subtitles:** Our automated software converts your file to text in minutes. If you choose the human service, your transcript will be ready within 24 hours.

6. **Select "Translation" and Choose Language:** Our video translator will generate the translation within a minute.

7. **Click "Export" and Choose File Format:** Easily get your translated video file.

https://github.com/ai-collection/ai-collection/assets/11485233/38eb06c0-16e8-4eb6-96bb-a53317a2d281

**Comment by adityapawar2004** [comment]
I had used VideoDubber.ai to translate my video, and I'm impressed!
Highly recommend!
